node/service/src/lib.rs

```diff
@@ -161,8 +162,7 @@ where
 		+ Send
 		+ Sync
 		+ 'static,
-	RuntimeApi::RuntimeApi:
-		RuntimeApiCollection<StateBackend = sc_client_api::StateBackendFor<FullBackend, Block>>,
```

Comment:
This change comes from substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)